src/NuGet.Core/NuGet.Protocol.Core.v3/ReportAbuseResourceV3.cs

```diff
@@ -32,50 +39,10 @@ public ReportAbuseResourceV3()
         /// <returns>The first URL from the resource, with the URI template applied.</returns>
         public Uri GetReportAbuseUrl(string id, NuGetVersion version)
         {
-            //return Utility.ApplyPackageIdVersionToUriTemplate(_reportAbuseTemplates.First(), id, version);
-            return new Uri(String.Format(CultureInfo.InvariantCulture, "https://www.nuget.org/packages/{0}/{1}/ReportAbuse", id, version.ToNormalizedString()));
-        }
+            var url = string.Format(CultureInfo.InvariantCulture, _uriTemplate, id, version.ToNormalizedString());
```

Comment:
That's bad. Updated in https://github.com/NuGet/NuGet.Client/commit/961ca49528aa2a666bae9f9c7361b08d2e06404d